app/models/concerns/hostext/search.rb

```diff
@@ -8,6 +8,7 @@ module Search
 
       has_many :search_parameters, :class_name => 'Parameter', :foreign_key => :reference_id
       belongs_to :search_users, :class_name => 'User', :foreign_key => :owner_id
+      belongs_to :usergroups, :class_name => 'Usergroup', :foreign_key => :owner_id
```

Comment:
Should this be rather singular? Or perhaps to be consisten with line above, the name should be search_usergroups?

Reply:
This is association to Usergroup model - is necessary to add it to audit_extension like line above ?

Reply:
nope, this is just another relation using owner_id, I think we already track owner_id and owner_type in audits